static/src/javascripts-legacy/projects/common/modules/experiments/ab-test-clash.js

```diff
@@ -26,9 +26,9 @@ define([
         name: 'ContributionsEpicRegulars',
         variants: ['control', 'fairness_mild', 'fairness_strong', 'fairness_strong_alternate_hook', 'reliance']
 	};
-    var AcquisitionsEpicDesignVariations = {
-        name: 'AcquisitionsDesignVariations',
-        variants: ['control', 'extra_paragraph', 'large_hook', 'subtle', 'prominent', 'highlight']
+    var AcquisitionsEpicDesignVariationsV2 = {
+        name: 'AcquisitionsDesignVariationsV2',
+        variants: ['control','subtle_highlight']
```

Comment:
nitpick: where did the spaaaaace go?

`variants: ['control', 'subtle_highlight']`